make/tests

```diff
@@ -85,12 +85,15 @@ HEADER_TESTS := $(addsuffix -test,$(call findfiles,src/stan,*.hpp))
 
 ifeq ($(OS),Windows_NT)
   DEV_NULL = nul
+  ifeq ($(IS_UCRT),true)
+    UCRT_NULL_FLAG = -S
```

Comment:
What does this flag do?

Reply:
It's an issue with the UCRT toolchain recognising the `nul` device, we previously discussed it in this [Math library PR](https://github.com/stan-dev/math/pull/2948#discussion_r1340248073)

Reply:
Huh, guess that one slipped my memory. Sounds good!